### PR TITLE
LPS-0 Avoid CXF logging redirections

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

File: modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/authorize/OAuthAuthorizationDataMessageBodyWriter.java [hunk 1]
```diff
@@ -92,6 +96,15 @@ public boolean isWriteable(
...
 
+	@Override
+	public Response toResponse(RedirectionException redirectionException) {
+		return Response.status(
+			Response.Status.FOUND
+		).location(
+			redirectionException.getLocation()
+		).build();
+	}
+
```
Comment (topolik) [inline modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/authorize/OAuthAuthorizationDataMessageBodyWriter.java]: Where is this used?

Comment (topolik) [reply modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/authorize/OAuthAuthorizationDataMessageBodyWriter.java]: Ah, sorry, I see, it converts the exception into status.

Comment (csierra) [reply modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/authorize/OAuthAuthorizationDataMessageBodyWriter.java]: yup

**Comment by topolik** [comment]
Already fixed.

**Pull request closed without merging**